code-challenges.test.js

```diff
@@ -14,21 +14,157 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibonacciLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibonacciLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// describe ("fiboSeq", () => {
+//   it ("takes in number < 2 and and returns an array containing the Fibonacci sequence with length determined by number", () => {
+    
+//     const fibonacciLength1 = 6
+//     expect [1, 1, 2, 3, 5, 8]
+//     // const numb1 = 6
+//     // expect [1, 1, 2, 3, 5, 8]
+//     const fibonacciLength2 = 10
+//     expect[1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+//     // const numb2 = 10
+// // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+//   })
+// })
+
+//Output: PASS - (thought the Red phase were supposed to produce one fail)
```

Comment:
You are correct, you should see the test fail first. It doesn't look like you have any Jest expect statements so I am not sure there was anything prompting your test to fail. It would be worth looking at the syllabus to ensure you have the correct syntax for your test.